### ISSUE TITLE
Color components and elements differently

### ISSUE THREAD

**Issue by TrySound** [mxw/vim-jsx#135]
vscode introduced this feature
https://twitter.com/mattbierner/status/861630591124381696
Would be good to see it in this project
The difference is upper and lower case of first letter.

**Comment by mxw** [comment]
Don't components and tags both have conceptually the same role?  Moreover—are components actually required to be uppercase?

**Comment by TrySound** [comment]
Yep. Lowercase is builtin primitive like tags, which are passed as strings to createElement (in context of react). Uppercase is custom component passed as class or component. The use case is to differ builtin primitives and user components.

**Comment by hdon** [comment]
I don't mind them having the same color.
